mail/template.invitation

```diff
@@ -0,0 +1,8 @@
+Hello {{.FirstName}} {{.LastName}},
+
+As part of the application process of {{.Company}}, you're invited to do a quick coding Challenge.
+
+You can access it via this link https://cod.uno/codeground/{{encode .Token}} and there you will find all the needed information.
```

Comment:
[https://cod.uno/](https://cod.uno/) does not work as expected, please use https://www.cod.uno/... for now.